src/libraries/Microsoft.PowerFx.Core/Public/Config/ComposedReadOnlySymbolTable.cs

```diff
@@ -19,12 +19,12 @@ namespace Microsoft.PowerFx
     /// </summary>
     internal class ComposedReadOnlySymbolTable : ReadOnlySymbolTable, INameResolver, IGlobalSymbolNameResolver, IEnumStore
     {
-        private readonly IEnumerable<ReadOnlySymbolTable> _symbolTables;
+        private readonly List<ReadOnlySymbolTable> _symbolTables;        
```

Comment:
>List [](http://example.com/codeflow?start=25&length=4)

do we need this change?

Reply:
little perf improvement iirc but not stictly needed